### ISSUE TITLE
[Property Explorer] Not all values in radio button groups are focused in Property Explorer when navigating with a keyboard 

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#701]
# Description
When User navigates with a keyboard, he can not get to all changeable properties.

# [Accordion component]
# Case №1

## Steps to Reproduce
1. Open [Accordion Property Explorer](https://uui.epam.com/documents?category=components&id=accordion&mode=propsEditor&skin=UUI3_loveship)
2. Navigate to the `title` radio group with a keyboard using `Tab` key.
3. Select `none` value.

## Actual result

User can not select `title=none` values.

https://user-images.githubusercontent.com/57751283/191027507-0ec29e1c-8ed8-4a88-9b3a-237bb1fbbb74.mp4

## Expected result

User can select the same values with the keyboard and with the mouse.

### Environment

* UUI version: 4.8.5
* Browser name: Chrome (ver. 105.0.5195.102); Microsoft Edge (ver. 05.0.1343.42).
* Operating System: Win 10

# [Alert component]
# Case №2

## Steps to Reproduce
1. Open [Accordion Property Explorer](https://uui.epam.com/documents?category=components&id=alert&mode=propsEditor&skin=UUI3_loveship)
2. Navigate to the `icon -> select icon` property with a keyboard using `Tab` key.
3. Press `Enter` to expand the list with icons.
4. Press `Up`/`Down` key to select an icon.
5. Press `Enter` key to confirm your selection.

## Actual result

Pop-up window with a list of icon doesn't close after selecting an icon.

https://user-images.githubusercontent.com/57751283/191511346-12c18e3d-1771-43e5-9ea5-79b3234a4585.mp4

## Expected result

Pop-up window with a list of icons closes when an icon is selected.

### Environment

* UUI version: 4.8.5
* Browser name: Chrome (ver. 105.0.5195.102); Microsoft Edge (ver. 05.0.1343.42); Firefox (105.0).
* Operating System: Win 10

# Case №3 - separate ticket

## Steps to Reproduce
1. Open [Alert Property Explorer](https://uui.epam.com/documents?category=components&id=alert&mode=propsEditor&skin=UUI3_loveship)
2. Navigate to the `color` radio group with a keyboard using `Tab` key.
3. Select any color (not `none`) value.

## Actual result

User can select only `color=none` value.

https://user-images.githubusercontent.com/57751283/191751743-c3f9f489-5b5a-4cb1-8b0a-820bb9ce41f0.mp4

## Expected result

User can select any color from the available.

### Environment

* UUI version: 4.8.5
* Browser name: Chrome (ver. 105.0.5195.102); Microsoft Edge (ver. 05.0.1343.42); Firefox (105.0).
* Operating System: Win 10

# [Anchor component]
# Case №4

## Steps to Reproduce
1. Open [Anchor Property Explorer](https://uui.epam.com/documents?category=components&id=anchor&mode=propsEditor&skin=UUI3_loveship)
2. Navigate to the `target` radio group with a keyboard using `Tab` key.
3. Select `_blank` value.

## Actual result

User can not select `target=_blank` value.

https://user-images.githubusercontent.com/57751283/191852339-c558c9c7-4ca2-4f14-b0b2-b0dea6676438.mp4

## Expected result

User can select the same values with the keyboard and with the mouse.

### Environment

* UUI version: 4.8.5
* Browser name: Chrome (ver. 105.0.5195.102); Microsoft Edge (ver. 05.0.1343.42); Firefox (105.0).
* Operating System: Win 10

# [Badge component]
# Case №5

## Steps to Reproduce
1. Open [Badge Property Explorer](https://uui.epam.com/documents?category=components&id=badge&mode=propsEditor&skin=UUI3_loveship)
2. Navigate to the `isDropdown` radio group with a keyboard using `Tab` key.
3. Select `true` value.

## Actual result

User can not select `isDropdown=true` value.

https://user-images.githubusercontent.com/57751283/191926486-47449d53-85a5-4711-8b46-5b99042b0061.mp4

## Expected result

User can select the same values with the keyboard and with the mouse.

## Note
User also can not select values in next radio groups:
- color;
- isOpen;
- onClick;
- onClear;
- caption.

### Environment

* UUI version: 4.8.5
* Browser name: Chrome (ver. 105.0.5195.102); Microsoft Edge (ver. 05.0.1343.42).
* Operating System: Win 10

## The principle of finding defects in the next components is exactly the same as in the cases described above.
To simplify, I will further write the name of the component and props with which there are problems

## [Blocker](https://uui.epam.com/documents?category=components&id=blocker&mode=propsEditor&skin=UUI3_loveship)
- spacerHeight.

## [Button](https://ui-dev.epm-ppa.projects.epam.com/documents?category=components&id=button&mode=propsEditor&skin=UUI3_loveship)
- isDropdown;
- isOpen;
- isDisabled;
- onClear;
- link;
- isLinkActive;
- href;
- target;
- color;
- caption ('none' value).

## [Checkbox](https://uui.epam.com/documents?id=checkbox&mode=propsEditor&skin=UUI3_loveship&category=components)
- isReadonly;
- isInvalid;
- color;
- label ('none' value).

## [Checkbox Group](https://uui.epam.com/documents?id=checkboxGroup&mode=propsEditor&skin=UUI3_loveship&category=components)
- isInvalid.

## [Date Picker](https://uui.epam.com/documents?category=components&id=datePicker&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- isInvalid;
- value;
- placeholder;
- renderDay;
- isHoliday
- renderFooter.

## [Dropdown](https://uui.epam.com/documents?category=components&id=dropdown&mode=propsEditor&skin=UUI4_promo) Promo skin
- openOnClick;
- openOnHover;
- closeOnClickOutside;
- closeOnTargetClick.

## [Icon Button](https://uui.epam.com/documents?category=components&id=iconButton&mode=propsEditor&skin=UUI3_loveship)
- isDisabled;
- isInvalid;
- link;
- isLinkActive;
- href;
- target;
- color

## [Icon Container](https://uui.epam.com/documents?category=components&id=iconContainer&mode=propsEditor&skin=UUI3_loveship)
- color.

## [Labeled Input](https://uui.epam.com/documents?category=components&id=labeledInput&mode=propsEditor&skin=UUI3_loveship)
- isInvalid;
- isRequired;
- isOptional;
- validationMessage;
- color.

## [Link Button](https://uui.epam.com/documents?category=components&id=linkButton&mode=propsEditor&skin=UUI3_loveship)
- isOpen;
- isDisabled;
- onClear;
- color;
- link;
- isLinkActive;
- href;
- target.

## [MultiSwitch](https://uui.epam.com/documents?category=components&id=multiSwitch&mode=propsEditor&skin=UUI3_loveship)
- isDisabled;
- color.

## [Notification Card](https://uui.epam.com/documents?category=components&id=notificationCard&mode=propsEditor&skin=UUI3_loveship)
- color.

## [NumericInput](https://uui.epam.com/documents?category=components&id=numericInput&mode=propsEditor&skin=UUI3_loveship)
- placeholder;
- isReadonly;
- formatter (Promo skin).

## [Picker Input](https://uui.epam.com/documents?category=components&id=pickerInput&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- suffix;
- isSingleLine;
- isFoldedByDefault;
- renderNotFound;
- renderFooter (Loveship skin).

## [RadioGroup](https://uui.epam.com/documents?category=components&id=radioGroup&mode=propsEditor&skin=UUI3_loveship)
- isInvalid.

## [RadioInput](https://uui.epam.com/documents?category=components&id=radioInput&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- color (Loveship skin);
- isInvalid;
- label ("none" value).

## [RangeDatePicker](https://uui.epam.com/documents?category=components&id=rangeDatePicker&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- isInvalid;
- value (Promo skin);
- getPlaceholder;
- renderDay;
- disableClear;
- isHoliday.

## [Rating](https://uui.epam.com/documents?category=components&id=rating&mode=propsEditor&skin=UUI3_loveship)
- isInvalid;
- color (Loveship skin);
- hideTooltip;
- hint.

## [Search Input](https://uui.epam.com/documents?category=components&id=searchInput&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- isInvalid;
- isOpen;
- onCancel;
- mode (Promo skin).

## [Slider](https://uui.epam.com/documents?category=components&id=slider&mode=propsEditor&skin=UUI3_loveship)
- color;
- isDisabled;
- renderLabel.

## [SliderRating](https://uui.epam.com/documents?category=components&id=sliderRating&mode=propsEditor&skin=UUI3_loveship)
- isReadonly.

## [Spinner](https://uui.epam.com/documents?category=components&id=spinner&mode=propsEditor&skin=UUI4_promo)
- color.

## [Switch](https://uui.epam.com/documents?category=components&id=switch&mode=propsEditor&skin=UUI3_loveship)
- color (Loveship skin);
- isInvalid (Promo skin).

## [Tab Button](https://uui.epam.com/documents?category=components&id=tabButton&mode=propsEditor&skin=UUI3_loveship)
- isDropdown;
- isOpen;
- isDisabled;
- onClear;
- link;
- isLinkActive;
- href;
- target.

## [Tag](https://uui.epam.com/documents?category=components&id=tag&mode=propsEditor&skin=UUI3_loveship)
- color (Loveship skin);
- isDropdown;
- isOpen;
- onClick (Promo skin);
- onClear (Promo skin).

## [Text](https://uui.epam.com/documents?category=components&id=text&mode=propsEditor&skin=UUI3_loveship)
- color.

## [TextArea](https://uui.epam.com/documents?category=components&id=textArea&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- isInvalid.

## [Text Input](https://uui.epam.com/documents?category=components&id=textInput&mode=propsEditor&skin=UUI3_loveship)
- isDisabled;
- isReadonly;
- isInvalid;
- isOpen;
- suffix (Loveship skin);
- prefix (Promo skin);
- onCancel.

## [TextPlaceholder](https://uui.epam.com/documents?category=components&id=textPlaceholder&mode=propsEditor&skin=UUI4_promo)
- color.

## [TimePicker](https://uui.epam.com/documents?category=components&id=timePicker&mode=propsEditor&skin=UUI3_loveship)
- isReadonly;
- isInvalid.

## [Vertical Tab Button](https://uui.epam.com/documents?category=components&id=verticalTabButton&mode=propsEditor&skin=UUI3_loveship)
- isDropdown;
- isOpen;
- isDisabled;
- onClear;
- link;
- isLinkActive;
- href;
- target.

## General Note
There is the following pattern in all property explorers: focus only goes to the next radio group after the multiswitch. 
In other words, if a radio group with a multiswitch is followed by two simple radio groups (without a multiswitch), the focus goes only to the first of the two, thus skipping the second one.

https://user-images.githubusercontent.com/57751283/194113092-adfd8852-dce1-48a6-8536-3070fe6d2527.mp4

**Comment by NatalliaAlieva** [comment]
Released in 5.6.0 ver.
